doc/index.rst

```diff
@@ -72,7 +72,7 @@ base :class:`~.OrquestraDevice` class.
     An option for Orquestra, specifies the resources to be specified for each
     workflow step.
 
-``timeout=300``
+``timeout=600``
```

Comment:
Observed that an increased timeout is required when doing batched execution

Reply:
Thanks! I wonder why